src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1460,14 +1468,30 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
                 attach_acr,
                 headers,
                 no_wait)
-            return created_cluster
+            break
```

Comment:
May I ask why you change `return created_cluster` to `break`?

Reply:
Because I need to create a DCR-A after the cluster is created. The logic to return created_cluster was moved out of that loop. Now that you ask, I think it would be cleaner to move the DCR-A creation call into the for loop.